src/rules/no-default-alt-text.js

```diff
@@ -3,53 +3,37 @@
 // e.g. "Screenshot 2020-10-20 at 2 52 27 PM"
 // e.g. "Clean Shot 2020-10-20 @45x"
 // e.g. "image"
-const defaultMacOsScreenshotMarkdownRegex =
-  /^(Screen|Clean) ?[S|s]hot \d{4}-\d{2}-\d{2}/gi;
-const imageMarkdownRegex = /^image$/i;
+// e.g. "14352435"
```

Comment:
Not blocking: Do we want to keep the comments showing the common examples of screen shot default alt text patterns (lines 3 & 4)?